docs/source/installing.rst

```diff
@@ -55,3 +52,16 @@ For developers, you can install TimeChart in development mode::
     git clone https://github.com/slaclab/timechart.git
     cd timechart
     python setup.py develop
+
+
+If you want to install a conda installation package for TimeChart:
+
+    * First, you must have ``conda-build`` installed::
+
+        conda install conda-build
+
+    * Then::
+
+        git clone https://github.com/slaclab/timechart.git
+        cd timechart
+        conda build -q conda-recipe --python=3.6 --output-folder bld-dir -c conda-forge -c pydm-tag -c conda-forge
```

Comment:
It is important to not in here since you will teach that ‘python=3.6’ must be changed to reflect the version that the developer is using.

Reply:
This section is missing the instruction to install the user created package. Just doing the conda-build does not install the package, it simply creates a tarballs that can be used to install. Check the instructions at the .travis.yml in which we do exactly that.